linode/manager
# test: [M3-8609] - Cypress test for non-empty Linode landing page with restricted user

## Description 📝
Added cypress test for non-empty Linode landing page with restricted user

## Changes  🔄
Added below cypress tests under in spec file - cypress/e2e/core/linodes/smoke-linode-landing-table.spec.ts
- checks restricted user with read access has no access to create linode and can see existing linodes


## PRs
- M3-8609

## How to test 🧪
`yarn cy:run -s cypress/e2e/core/linodes/smoke-linode-landing-table.spec.ts`

### Verification steps
When test executed using this command  `yarn cy:run -s cypress/e2e/core/linodes/smoke-linode-landing-table.spec.ts`all tests should pass successfully and you should see new test added as per below screenshot 

![image](https://github.com/user-attachments/assets/34eb0d5e-9969-462d-8ac1-b26eabdc6702)


## As an Author I have considered 🤔

*Check all that apply*
